src/views/EServiceManage.tsx

```diff
@@ -255,31 +231,39 @@ export function EServiceManage() {
         </Stack>
       </Stack>
 
-      {eserviceData && (
+      {!isLoading && eserviceData && descriptorId ? (
```

Comment:
why `descriptorId` instead of `mode==="provider"`?

Reply:
This line is part of the conditional render of the content/spinner based on the loading state. 

"**eserviceData && descriptorId**" is there just because I didn't want to put the non-null assertion operator in the props passed to the EServiceContentInfo, so I checked for them being non-null there.

It might not be clear, do you prefer to put the non-null assertion to the component's props?

```jsx
<EServiceContentInfo
  data={eserviceData!}
  descriptorId={descriptorId!}
  context="provider"
/>
```  

N.B: asserting that those variable are not null is correct in this context.